src/coreclr/debug/ee/controller.cpp

```diff
@@ -7820,6 +7893,17 @@ void DebuggerStepper::TriggerExternalMethodFixup(PCODE target)
     this->DisableExternalMethodFixup();
 }
 
+void DebuggerStepper::TriggerGenericPInvokeCalli(PCODE target)
+{
+    TraceDestination trace;
+    FramePointer fp = LEAF_MOST_FRAME;
+    trace.InitForStub(target);
+    g_pEEInterface->FollowTrace(&trace);
```

Comment:
Should we check the return value here and conditionally execute PatchTrace?

Reply:
In this case, FollowTrace should always return True. Should we assert that that is the case? Then, if there is an issue (or a case that I do not know about) we will find it easily.